src/test/java/com/pgs/client/interceptor/LoggingInterceptorTest.java

```diff
@@ -0,0 +1,101 @@
+package com.pgs.client.interceptor;
+
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.boot.test.system.CapturedOutput;
+import org.springframework.boot.test.system.OutputCaptureExtension;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpRequest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.client.ClientHttpRequestExecution;
+import org.springframework.http.client.ClientHttpResponse;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+import org.springframework.util.StreamUtils;
+
+import java.io.ByteArrayInputStream;
+import java.net.URI;
+import java.nio.charset.StandardCharsets;
+import java.util.Random;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.when;
+
+
+@ExtendWith({OutputCaptureExtension.class, MockitoExtension.class})
+class LoggingInterceptorTest {
+
+    @Mock
+    private HttpRequest httpRequest;
+    @Mock
+    private ClientHttpRequestExecution clientHttpRequestExecution;
+    @Mock
+    private ClientHttpResponse clientHttpResponse;
+    @InjectMocks
+    private LoggingInterceptor loggingInterceptor;
+
+    private static byte[] ARR = new byte[20];
```

Comment:
![image](https://user-images.githubusercontent.com/15009199/140946155-6ea47d6f-7c13-4e24-864a-da4abc9bb629.png)
give it a better name 😉